server/cmd/mattermost/commands/import.go

```diff
@@ -92,10 +92,10 @@ func slackImportCmdF(command *cobra.Command, args []string) error {
 	CommandPrettyPrintln("Finished Slack Import.")
 	CommandPrettyPrintln("")
 
-	auditRec := a.MakeAuditRecord("slackImport", audit.Success)
+	auditRec := a.MakeAuditRecord(request.EmptyContext(a.Log()), "slackImport", audit.Success)
```

Comment:
Let's move the request to it's own variable at the start of the method and reuse it for all methods that need a request.